### ISSUE TITLE
Column dropdown options should be configurable.

### ISSUE THREAD

**Issue by vincentcn** [predixdesignsystem/px-data-grid#783]
Current header actions in a data grid can be hidden by configuration
![header_actions](https://user-images.githubusercontent.com/443663/37911525-dd4d1558-3142-11e8-8e76-a173dd29b65a.png)

But I didn't find an API(prop) to hide/customize column dropdown options
![column_dropdown_options](https://user-images.githubusercontent.com/443663/37911454-ad098a84-3142-11e8-93e4-27b0586bb16e.png)

If the header actions is hidden while "Hide Column" still work in column dropdown actions. User cannot show a column again after he/she hide it via column dropdown action.

So suppose we should make column dropdown options configurable.

**Comment by evanjd** [comment]
Sorry for the slow reply to this.

> User cannot show a column again after he/she hide it via column dropdown action.

What do you mean? You can simply reselect the column in the dropdown after you've hidden it.

> If the header actions is hidden while "Hide Column" still work in column dropdown actions.

I'm not sure what you're saying - can you rephrase?

**Comment by ajjaskie** [comment]
We are running into this issue as well. We have the "Action" menu in the top right hidden. Users can still hide columns using the drop-down on each column. When they do so, they cannot un-hide them because the Action menu is hidden. We need to be able to hide the "Hide Column" option, or hide the column menu entirely.

Related issues:
#793 
#304
